lib/portage/gpkg.py

```diff
@@ -628,6 +628,10 @@ def extractall(self, dest_dir: str):
         if self.closed:
             raise OSError("Tar file is closed.")
         temp_dir = tempfile.TemporaryDirectory(dir=dest_dir)
+        try:
+            self.tar.extraction_filter = tarfile.fully_trusted_filter
+        except AttributeError:
+            pass
```

Comment:
We could refactor the security checks into a filter function that raises exceptions for violations. The `fully_trusted_filter` is simply an identity function like this:
```python
def fully_trusted_filter(member, dest_path):
    return member
```

Reply:
Yeah, that might be a nice idea and worth a TODO comment. Not a blocker though obviously.

Reply:
Comment added, thanks!